README.md

```diff
@@ -9,16 +9,16 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 ## Example
 
 ```js
+var superagent = require('superagent'),
+    superagentUse = require('superagent-use'),
+    prefix = require('superagent-prefix'); /* A sample superagent plugin/middleware. */
 
-var superagent = require('superagent');
-/* Pass the superagent object to superagent-use
-   in order to be provided with new `use` functionality */
-require('superagent-use')(superagent);
-var prefix = require('superagent-prefix')('https://api.example.com');
+/* superagentUse returns a clone of the superagent provided with the new functionality. */
+var request = superagentUse(superagent);
```

Comment:
Naming the variable `request` is misleading, because no request have been made yet. We should just keep it at `superagent`